emulators/qemu/Portfile

```diff
@@ -250,6 +280,27 @@ platform darwin 8 {
     build.cmd ${prefix}/bin/gmake
 }
 
+platform darwin 20 {
+    post-destroot {
+        # meson modify binary during install which broke codesign for hvf-enabled binaries
+        # we need to re-sign it again
+        set codesign_targets {}
+
+        if {${configure.build_arch} eq "x86_64"} {
```

Comment:
These blocks match the binaries created by the default variants. However, if any other `+target_*` variant was chosen, they must also be broken. Could we just sign anything matching a `qemu-system-*` wildcard?

For example:
```
foreach target [glob -directory "${destroot}${prefix}/bin/" "qemu-system-*"] {
    ...
}
```

Reply:
There seems now to be a patch to fix codesign without requiring to manually sign via MacPorts. I think we can use this instead: https://lore.kernel.org/qemu-devel/20210225000614.46919-1-akihiko.odaki@gmail.com/